### PR TITLE
feat(AcademicProfileCommentSchema): Add general as type comment

### PR DESCRIPTION
**Pull request by MaiaRojas**
Se añade un nuevo tipo de comentario.

### CODE REVIEW

**Comment by merycardenas27** [comment]
Great job :+1: 
@MaiaRojas , It would be really nice if you could add a description to this PR :smile:

File: src/AcademicProfileCommentSchema.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ module.exports = (conn) => {
       enum: [
         'tech',
         'soft',
+        'general',
```
Comment (Xpktro) [inline src/AcademicProfileCommentSchema.js:23]: Good, even though this is considered an `enum value` rather tan a property (which is `type` in this case). Please describe this in your commit message as well.

**Review by Xpktro — CHANGES_REQUESTED** [review]
What Mery said

**Review by merycardenas27 — APPROVED** [review]

**Pull request merged**